### DISCUSSION TITLE
How to change completion colors?

### DISCUSSION THREAD

**Discussion by Mediocris** [marlonrichert/zsh-autocomplete#345] [Q&A] [answered]
How to change the color of the directories shown below? Red color is too dazzling.
<img width="888" alt="截屏2021-09-26 下午4 15 16" src="https://user-images.githubusercontent.com/38163902/134799599-7001242c-9c2b-4011-9df0-0f5bc5555948.png">

**Comment by marlonrichert** [comment, marked as answer]
The easiest way is to use my [Z Colors](https://github.com/marlonrichert/zcolors) plugin.

**Reply by Mediocris** [reply]
I have installed Z Colors plugin and set the LSCOLORS=Gxfxcxdxbxegedabagacad, but the completion color is still red. 
<img width="887" alt="截屏2021-11-02 下午1 08 58" src="https://user-images.githubusercontent.com/38163902/139789894-369b1f05-ff45-49c5-b72e-1cef509a0e73.png">
<img width="887" alt="截屏2021-11-02 下午2 47 59" src="https://user-images.githubusercontent.com/38163902/139799105-d9a470a8-cd0f-42f0-be5f-19e6fcd03d49.png">

**Reply by marlonrichert** [reply]
> `LSCOLORS=Gxfxcxdxbxegedabagacad`

You need to set `$LS_COLORS`, with an underscore. You can use, for example, https://geoff.greer.fm/lscolors/ to generate the correct syntax. Notice how it is different from `$LSCOLORS`.

However, you can also call `zcolors` with passing any arguments. It has good defaults.

**Reply by marlonrichert** [reply]
I created an issue for adding `$LSCOLORS` support Z Colors: https://github.com/marlonrichert/zcolors/issues/15
